index.html

```diff
@@ -5,20 +5,28 @@
     <title>Integrative Coding Experience</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width,initial-scale-1.0" />
+    <link rel="stylesheet" href="style.css" />
   </head>
 
   <body>
-    <h1>I.C.E</h1>
-    <p>Hello World!</p>
-    <div id="code-block">
-      <textarea id="code-area"></textarea>
+    <div id="main-div">
```

Comment:
I would recommend using semantic HTML elements where possible. For example, instead of `<div id="main-div">` prefer to simply use `<main>`. Here's a page with more info about semantic HTML tags:

https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantic_elements